lab-dustin/test/test.js

```diff
@@ -0,0 +1,47 @@
+const assert = require('assert');
+
+const greet = require('../lib/greet.js');
+
+describe('Greet Fucntion', function() {
+  describe('#greetfunction()', function() {
+    it(`should return 'hello world!' when passed 'world!'`, function() {
+      assert.strictEqual(greet.greetFunction('world!'), 'hello world!');
+    });
+    it('should return null when function is not passed a string', function() {
+      assert.equal(greet.greetFunction(3), null);
+    });
+    it('should return null when function is not passed a string', function() {
+      assert.equal(greet.greetFunction(true), null);
+    });
+  });
+});
+//==================================//
+const operators = require('../lib/arithmetic.js');
+
+describe('Operator Functions', function(){
+  describe('#addFunction', function(){
+    it('should return 2 when passed params 1, 1', function(){
+      assert.strictEqual(operators.addFunction(1,1), 2);
+    });
+    it('should return null if passed non-number', function(){
+      assert.equal(operators.addFunction('3',3), null);
+    });
+    it('should return null if passed NaN', function(){
+      assert.equal(operators.addFunction(NaN, 3), null);
```

Comment:
Not necessary to change, but as discussed in class - a good idea to include a second `assert` here that checks handling of `NaN` as the second argument. Same for other invalid inputs, both in first and second position - a common source of bugs.